### PR TITLE
Documentation Additions

### PR DESCRIPTION
**Pull request by ben-directai**
Summary:
Adding explanation to closely match [viam's motion detector](https://github.com/viam-labs/motion-detector/) docs.

See rendering of README [here](https://github.com/DirectAI/ViamIntegration/tree/docs_update).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,41 +3,136 @@ Module of the Viam vision service to automatically build, deploy, and run infere
...
 }
 ```
+
+## Visualize
+Once the `directai:viamintegration:directai-beta` modular service is in use, configure a [transform camera](https://docs.viam.com/components/camera/transform/) to see classifications or detections appear in your robot's field of vision. View the transform camera from the [Control tab](https://docs.viam.com/manage/fleet/robots/#control).
+
+## Next Steps
+To write code to use the motion detector output, use one of the [available SDKs](https://docs.viam.com/program/).
+
+Please join [DirectAI's discord](https://discord.com/invite/APU6MWBKQv), contact us at [ben@directai.io](mailto:ben@directai.io), or schedule time on our calendly if you have any questions or feedback!
```
Comment (isaacrob) [inline README.md:133]: no calendly link?

File: README.md [hunk 2]
```diff
@@ -3,41 +3,136 @@ Module of the Viam vision service to automatically build, deploy, and run infere
...
 
 ### Attributes
 
 The following attributes are available for `directai:viamintegration:directai-beta` vision services:
+| Name | Type | Inclusion | Description |
+|------|------|-----------|-------------|
+| `access_json` | string | **Required** | A string that indicates an absolute path on your local machine to a JSON including DirectAI Client Credentials. See description in [Example Access JSON](#example-access-json) section. |
+| `deployed_classifier` | json | **Optional** | A JSON that contains a `classifier_configs` key and corresponding list of classifier configurations. Each classifier is defined by a `name`, a list of text `examples_to_include`, and a list of text `examples_to_exclude`. See [Example Classifier Attributes](#example-classifier-attributes). |
+| `deployed_detector` | json | **Optional** | A JSON that contains a `detector_configs` key and corresponding list of detector configurations. Each detector is defined by a `name`, a list of text `examples_to_include`, and a list of text `examples_to_exclude`. See [Example Detector Attributes](#example-detector-attributes). |
```
Comment (isaacrob) [inline README.md:98]: detection threshold / nms threshold?

File: README.md [hunk 3]
```diff
@@ -3,41 +3,136 @@ Module of the Viam vision service to automatically build, deploy, and run infere
...
+  "DIRECTAI_CLIENT_SECRET": "L23LKkl0d5<M0R3S4F3"
+}
+```
+
 ## Configuration
 
 Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Services** subtab and click **Create service**. Select the `vision` type, then select the `directai-beta` model. Enter a name for your service and click **Create**.
 
-### Example Configuration
+On the new component panel, copy and paste the Example Detector *or* Classifier Attributes. Ensure that the Access JSON path that you provide in your Config is **absolute**, not relative. (e.g. `/Users/janesmith/Downloads/directai_credential.json`, not `~/Downloads/directai_credential.json`)
```
Comment (isaacrob) [inline README.md:26]: should be able to do both detector and classifier in one json

**Review by isaacrob — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ben-directai) [reply README.md:133]: thanks good catch

File: README.md (hunk 2 above)
Comment (ben-directai) [reply README.md:98]: thanks good catch

File: README.md (hunk 3 above)
Comment (ben-directai) [reply README.md:26]: I opted to show one at a time for simplicity but I can make a mention of it in the description. happy to discuss more if need be

File: README.md (hunk 1 above)
Comment (ben-directai) [reply README.md]: update: fixed

File: README.md (hunk 2 above)
Comment (ben-directai) [reply README.md]: update: fixed

File: README.md (hunk 3 above)
Comment (ben-directai) [reply README.md]: update: made a note

**Review by isaacrob — APPROVED** [review]
lgtm

**Pull request merged**